client/me/profile/updated-gravatar-string.js

```diff
@@ -0,0 +1,32 @@
+import { englishLocales } from '@automattic/i18n-utils';
+import i18n, { localize } from 'i18n-calypso';
+import { Component } from 'react';
+
+class UpdatedGravatarString extends Component {
+	render() {
+		const { translate, locale, gravatarProfileLink } = this.props;
+		const stringParts = {
+			components: {
+				spanLead: <strong className="profile__link-destination-label-lead" />,
+				spanExtra: <span className="profile__link-destination-label-extra" />,
+				profileLink: <a href={ gravatarProfileLink } target="_blank" rel="noreferrer" />,
+				deleteLink: (
+					<a href="https://gravatar.com/account/disable/" target="_blank" rel="noreferrer" />
+				),
+			},
+		};
+		return englishLocales.includes( locale ) ||
+			i18n.hasTranslation(
+				'{{spanLead}}Hide my photo and Gravatar profile.{{/spanLead}} {{spanExtra}}This will prevent your photo and {{profileLink}}Gravatar profile{{/profileLink}} from appearing on any site. It may take some time for the changes to take effect. Gravatar profiles can be deleted at {{deleteLink}}Gravatar.com{{/deleteLink}}.{{/spanExtra}}'
+			)
+			? translate(
+					'{{spanLead}}Hide my photo and Gravatar profile.{{/spanLead}} {{spanExtra}}This will prevent your photo and {{profileLink}}Gravatar profile{{/profileLink}} from appearing on any site. It may take some time for the changes to take effect. Gravatar profiles can be deleted at {{deleteLink}}Gravatar.com{{/deleteLink}}.{{/spanExtra}}',
+					stringParts
+			  )
```

Comment:
Could we extract these strings, that are identical, to a constant?

Reply:
> Could we extract these strings, that are identical, to a constant?

@zaguiini Unfortunately, I tried that, but `translate()` doesn't like to receive a constant.